doc/source/index.rst

```diff
@@ -21,5 +21,5 @@ Key provided features:
    utilities
    design
    developer
-   api
```

Comment:
But I am used to seeing API at the top of the toc in other documentation, and 'developer' section is meant to contain more content than just a link to API.  Maybe it is just me :/